modules/account/src/main/kotlin/edu/stanford/spezi/module/account/manager/UserSessionManager.kt

```diff
@@ -21,6 +21,8 @@ interface UserSessionManager {
     suspend fun getUserState(): UserState
     fun observeUserState(): Flow<UserState>
     fun getUserUid(): String?
+    fun getUserName(): String?
+    fun getUserEmail(): String?
 }
```

Comment:
As proposed in a previous comment, let introduce a new type `UserInfo` and just expose it via a single method